site/guide/configuration/personalize-validmind.qmd

```diff
@@ -0,0 +1,18 @@
+---
+title: "Personalize ValidMind"
```

Comment:
It feels like we have quite a few verbs kicking around in these topics that essentially describe very similar actions. Perhaps do light clean-up pass to see what could be simplified? 

E.g. there's 

- personalize
- customize
- configure
- view (but is actually also manage/enable/etc.)

and these often get used interchangeably within the same topic. 

I don't think terminology needs to be 100% consistent, but it's not a bad thing to use repetitive phrasing for a global audience.

Reply:
Good point! I was thinking about how we should handle these, as now that that we have a "section" that might be expanded on in the future it's good to be consistent. I have made the following changes for now:

<img width="1476" alt="Screenshot 2024-08-07 at 2 30 26 PM" src="https://github.com/user-attachments/assets/f59f382d-8bba-4d5b-9a76-876d9d343dd2">